src/modules/RootPart.rb

```diff
@@ -2319,6 +2306,41 @@ def update_var_dev_name(var_name, fstab, root_current_name)
       partition = other_disk.partitions.find { |part| part.number == var_name_number.to_i }
       partition.name
     end
+
+    # @see #mount_regular_fstab_entry?(
+    ALLOWED_FS = [
+      "ext",
+      "ext2",
+      "ext3",
+      "ext4",
+      "btrfs",
+      "jfs",
+      "xfs",
+      "hpfs",
+      "vfat",
+      "auto",
+    ]
+    private_constant :ALLOWED_FS
+
+    # Whether a given fstab entry should be mounted by {#MountFSTab}
+    #
+    # @param entry [Hash] fstab entry
+    # @return [Boolean]
+    def mount_regular_fstab_entry?(entry)
+      vfstype = entry.fetch("vfstype", "")
+      mntops = entry.fetch("mntops", "")
+      path = entry.fetch("file", "")
+
+      return false if path == "/"
+      return false unless ALLOWED_FS.include?(vfstype)
+      return false if mntops.include?("noauto")
+      # The conditions above are enough for any mount point except /var
+      return true if path != "/var"
+
+      # Condition for /var. In general, it should have been already processed by
+      # #MountVarIfRequired... except when /var is a subvolume
+      !IsMounted("/var") && mntops.include?("subvol=")
```

Comment:
This `&& mntops...` gives me headaches. Whatever you say, if `!IsMounted()` doesn't do its job, this looks illogical to me.

Reply:
Or maybe I'm missing something.

Reply:
As commented by Stefan, should not it be mounted when is not mounted, independently whether it is a subvolume or not?

Reply:
I just wanted to keep the old logic about IsMounted which, you know what? It's based on a variable `@activated` that is not updated for everything that is mounted, just for some things in some situations (I don't didn't dare to check which ones and/or why). So I kept the condition just in case.

On the other hand, the method was broken and always returning true, so maybe there is no point in keeping the call to it.

Reply:
In other words, `IsMounted` doesn't do what you would have expected.

Reply:
Then maybe its name is wrong. But as it stands, that line looks weird.

Reply:
I will remove the `IsMounted` part. It's pointless since in the past it was broken anyways so it has never added any value.